### DISCUSSION TITLE
Higher Display Resolution than 1024*768

### DISCUSSION THREAD

**Discussion by runningdave** [ubenmackin/ACVM#28] [General]
I have installed the viogpudo drivers, hoping that i can use them to change my display resolution to a higher resolution than 1024. But I cannot open the system settings in the settings app to change the display setting. when I press 'system settings' the settings app alwas freezes. Is this normal? I have installed the latestes system updates (version 21277 - 2011207-1443)

**Comment by ubenmackin** [comment]
@runningdave this doesn't happen for me. The Store app works fine, and I can change the resolution.

That said, I did have an issue my first go around, using the pre-made VHDX image. After a few reboots, it would get corrupt, apps would stop working, and then eventually it would fail to boot. Apparently this is an issue for a number of users.

A number of people recommended converting the drive from VHDX is qcow2, and that fixed the corruption issues:

https://forums.macrumors.com/threads/success-virtualize-windows-10-for-arm-on-m1-with-alexander-grafs-qemu-hypervisor-patch.2272354/post-29384878

Note that this doesn't help if you are already corrupted. But if starting from scratch, it is recommended to do this from the beginning.

**Comment by runningdave** [comment]
@ubenmackin, thanks for the reply. I have converted my image to qcow2, but not right from the beginning. VHDX didn't work for me at all, after 2-3 reboots the images always failed to boot. I'll try tomorrow to convert the original download and start from scratch.

**Reply by runningdave** [reply]
Converting the VHDX to qcow2 before using it at all, has solved my problems. I can now use higher resolutions with the viogpudo drivers. But I have not installed the DEV branch update 21277 again.  So I can't say for shure, that it wasn't the update that has caused the problems.
